src/prefect/server/database/orm_models.py

```diff
@@ -875,6 +875,7 @@ def job_variables(self):
         order_by=sa.desc(sa.text("updated")),
     )
 
+    concurrency_limit = sa.Column(sa.Integer, default=None, nullable=True)
```

Comment:
With sqlalchemy2 we can actually have correct typing by using `Mapped`:

```suggestion
    concurrency_limit: Mapped[int] = mapped_column(sa.Integer, default=None, nullable=True)
```